easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.63.1-GCCcore-8.3.0-Python-3.7.4.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'MesonNinja'
+
+name = 'GObject-Introspection'
+version = '1.63.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.gnome.org/GObjectIntrospection/'
+description = """GObject introspection is a middleware layer between C libraries
+ (using GObject) and language bindings. The C library can be scanned at
+ compile time and generate a metadata file, in addition to the actual
+ native C library. Then at runtime, language bindings can read this
+ metadata and automatically provide bindings to call into the C library."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['476379bde8d080d92dd1bb1684f6aa8d79d90ddb81fc85dfa3576e36f9777ab6']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('Meson', '0.51.2', '-Python-3.7.4'),
+    ('Ninja', '1.9.0'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.3.2'),
+    ('cairo', '1.16.0'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('GLib', '2.62.0'),
+    ('libffi', '3.2.1'),
+    ('util-linux', '2.34'),
+]
+
+preconfigopts = "env GI_SCANNER_DISABLE_CACHE=true "
+
+# avoid using hard-coded path to 'python3' in shebang of scripts
+# preconfigopts += "PYTHON=python3 "
+# buildopts = "PYTHON=python3 "
+
+modextrapaths = {
+    'GI_TYPELIB_PATH': 'share',
```

Comment:
As per discussion in another thread, GI_TYPELIB_PATH should be os.path.join(x, 'girepository-*') for x in lib_paths

And both GI_TYPELIB_PATH and XDG_DATA_DIRS is now handled by the framework (PR 3133)

And as per easyconfig PR 9528 the modextrapaths for all packages that has these two has been removed.

So please remove that part.

Reply:
Removed in latest commit